src/winter_onboarding_2021/fleet_management_service/handlers/fleet.clj

```diff
@@ -0,0 +1,32 @@
+(ns winter-onboarding-2021.fleet-management-service.handlers.fleet
+  (:require [winter-onboarding-2021.fleet-management-service.models.fleet :as fleet-model]
+            [clojure.spec.alpha :as s]
+            [winter-onboarding-2021.fleet-management-service.specs :as specs]
+            [ring.util.response :as response]
+            [winter-onboarding-2021.fleet-management-service.views.fleet :as views]))
+
+(def success-flash
+  {:flash {:success true
+           :style-class "alert alert-success"
+           :message "Fleet created successfully!"}})
+(def error-flash
+  {:flash {:error true
+           :style-class "alert alert-danger"
+           :message "Could not create fleet, try again!"}})
+
+(defn create-fleet [req]
+  (let [user-id (get-in req [:user :users/id])
+        fleet-name (get-in req [:form-params :name])
+        fleet-data {:name fleet-name
+                    :created-by user-id}
+        valid-fleet-data (s/conform ::specs/create-fleet fleet-data)]
```

Comment:
There's no need to bind to valid-fleet-data. You can replace it with the conform call in place, since it is used only once and it is quite  clear as to what it is doing.